guides/source/action_text_overview.md

```diff
@@ -163,24 +146,66 @@ To customize the HTML rendered for embedded images and other attachments (known
       <span class="attachment__name"><%= blob.filename %></span>
       <span class="attachment__size"><%= number_to_human_size blob.byte_size %></span>
     <% end %>
-  </figcaption>
-</figure>
+  </figcapt
 ```
 
-### Rendering attachments
+## Attachments
+
+Currently ActionText supports attachments that are uploaded through Active Storage as well as attachments that are linked to a Signed GlobalId.
+
+### Active Storage
+
+When uploading an image within your rich text editor, it uses Action Text which in turn uses Active Storage. However, Active Storage has some dependencies which are not provided by Rails. To use the built-in previewers, you must install the following:
+
+- (required) [libvips](https://github.com/libvips/libvips) v8.6+ or [ImageMagick](https://imagemagick.org/index.php) for image analysis and transformations like resizing an image
+- [ffmpeg](http://ffmpeg.org/) v3.4+ for video previews and ffprobe for video/audio analysis
+- [poppler](https://poppler.freedesktop.org/) or [muPDF](https://mupdf.com/) for PDF previews
+
+You should familiarize yourself with the licensing implications of installing the above. Some, but not all of these libraries are required and they are dependent on the kind of uploads you are expecting within the editor. A common error that users encounter when working with Action Text and Active Storage is that images do not render correctly in the editor. This is usually due to the `libvips` dependency not being installed.
+
+
+### Signed GlobalId
+
+In addition to attachments uploaded through Active Storage, Action Text can also embed anything that can be resolved by a Signed GlobalID.
+
+A Global ID is an app wide URI that uniquely identifies a model instance: `gid://YourApp/Some::Model/id`. This is helpful when you need a single identifier to reference different classes of objects.
+
+When using this method, ActionText requires attachments to have a signed global ID (sgid). By default, all ActiveRecord models in a Rails app mix-in `GlobalID::Identification` concern. So they can be resolved by a signed global ID and are therefore `ActionText::Attachable` compatible.
 
-In addition to attachments uploaded through Active Storage, Action Text can
-embed anything that can be resolved by a [Signed
-GlobalID](https://github.com/rails/globalid#signed-global-ids).
+Action Text minifies the HTML you insert on save, so that it can re-render it with up-to-date content later on. This makes it so that you can reference models and always display the current content when those records change.
```

Comment:
I'm slightly afraid the word `minifies` here might lead some people to think the whole HTML from action text gets somehow minified to be stored. I think the intention is to convey that it will reference models as `<action-text-attachment>` tags pointing to their `attachable_sgid`, rather than embedding the final full rendered HTML of that referenced attachment, right?

I'm not sure I have a good suggestion off the top of my head, but I do think we should try to avoid the "minifying" concept. I think we're looking for something like "references" or similar. A couple paragraphs below this becomes a bit clearer.

Perhaps a quick re-read through the whole section once other feedback has been reviewed will show some light on how we can potentially tweak it here. Let me know if you have any thoughts.

Reply:
I've changed it to "references" for now. I'm going to continue thinking about this.